examples/src/main/java/ChangeRemoveTokenKeys.java

```diff
@@ -0,0 +1,150 @@
+import com.hedera.hashgraph.sdk.AccountId;
+import com.hedera.hashgraph.sdk.Client;
+import com.hedera.hashgraph.sdk.KeyList;
+import com.hedera.hashgraph.sdk.PrivateKey;
+import com.hedera.hashgraph.sdk.PublicKey;
+import com.hedera.hashgraph.sdk.TokenCreateTransaction;
+import com.hedera.hashgraph.sdk.TokenInfoQuery;
+import com.hedera.hashgraph.sdk.TokenKeyValidation;
+import com.hedera.hashgraph.sdk.TokenType;
+import com.hedera.hashgraph.sdk.TokenUpdateTransaction;
+import io.github.cdimascio.dotenv.Dotenv;
+import java.util.Objects;
+
+public class ChangeRemoveTokenKeys {
+
+    // see `.env.sample` in the repository root for how to specify these values
+    // or set environment variables with the same names
+    private static final AccountId OPERATOR_ID = AccountId.fromString(Objects.requireNonNull(Dotenv.load().get("OPERATOR_ID")));
+
+    private static final PrivateKey OPERATOR_KEY = PrivateKey.fromString(Objects.requireNonNull(Dotenv.load().get("OPERATOR_KEY")));
+
+    // HEDERA_NETWORK defaults to testnet if not specified in dotenv
+    private static final String HEDERA_NETWORK = Dotenv.load().get("HEDERA_NETWORK", "testnet");
+
+    private ChangeRemoveTokenKeys() {
+    }
+
+    public static void main(String[] args) throws Exception {
+        Client client = ClientHelper.forName(HEDERA_NETWORK);
+
+        // Defaults the operator account ID and key such that all generated transactions will be paid for
+        // by this account and be signed by this key
+        client.setOperator(OPERATOR_ID, OPERATOR_KEY);
+
+        // Admin, Supply, Wipe keys
+        var adminKey = PrivateKey.generateED25519();
+        var supplyKey = PrivateKey.generateED25519();
+        var newSupplyKey = PrivateKey.generateED25519();
+        var wipeKey = PrivateKey.generateED25519();
+
+        // This HIP introduces ability to remove lower-privilege keys (Wipe, KYC, Freeze, Pause, Supply, Fee Schedule, Metadata) from a Token:
+        // - using an update with the empty KeyList;
+        var emptyKeyList = new KeyList();
+        // - updating with an “invalid” key such as an Ed25519 0x0000000000000000000000000000000000000000000000000000000000000000 public key,
+        // since it is (presumably) impossible to find the 32-byte string whose SHA-512 hash begins with 32 bytes of zeros.
+        var unusableKey = PublicKey.fromString("0000000000000000000000000000000000000000000000000000000000000000");
```

Comment:
Might be good idea to define a public const `ZERO_KEY_STRING` and use it in the example and in the tests.
See the go implementation for example : https://github.com/hashgraph/hedera-sdk-go/pull/920/files#diff-17a4c9ccab09d27d5b16120cdef166d735de80ff2b2f86de36c031ab90cce652

Reply:
I like this idea! @agadzhalov wdyt?
(I just would like it to be consistent across all of the SDKs)

Reply:
I agree, we can use this approach 🚀

Reply:
Done